modules/fedimint-ln/src/lib.rs

```diff
@@ -342,6 +345,83 @@ impl ModuleGen for LightningGen {
     ) -> anyhow::Result<bitcoin_hashes::sha256::Hash> {
         serde_json::from_value::<LightningClientConfig>(config)?.consensus_hash()
     }
+
+    async fn dump_module_database(
+        &self,
+        dbtx: &mut DatabaseTransaction<'_>,
+        prefixes: Vec<String>,
```

Comment:
Should we use `Vec<u8>` for prefixes?

Reply:
It's actually just a string that refers to a specific prefix (e.g "Accepted Transaction"). Figured it was easier to write on the command line than translating what the actual prefix is.

Maybe column is a better name?

Reply:
Changed to prefix_names